README.md

```diff
@@ -30,23 +30,21 @@ $ go get github.com/milosgajdos83/servpeek
 Example test file could look like this:
 
 ```go
-package examples
+package example
 
 import (
 	"testing"
 
-	"github.com/milosgajdos83/servpeek/resource"
-	"github.com/milosgajdos83/servpeek/resource/pkg"
+	"github.com/milosgajdos83/servpeek/pkg"
 )
 
 func TestGemPackage(t *testing.T) {
-	testPkg := resource.Pkg{
-		Name:    "bundler",
-		Version: "1.10.6",
-		Type:    "gem",
+	testPkg, err := pkg.NewSwPkg("gem", "bundler", "1.10.6")
```

Comment:
I was trying to see how to get rid of `err` but I couldn't figure anything out.

Do you think that it should be on the `NewSwPkg` or in other check as `IsInstalled`?